README.md

```
@@ -72,6 +72,42 @@ To import only device by APC, for example:
 ./nb-dt-import.py --vendors apc,juniper
 ```
 
+#### Update Mode
+
+By default, the script only creates new device types and skips existing ones. To update existing device types:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)